views/about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Gavin Wood </title>
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" 
+    rel="stylesheet" 
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" 
+    crossorigin="anonymous">
+
+    <link rel="stylesheet" href="/public/stylesheets/styles.css">
+
+</head>
+<body style="background-color: darkgray;">
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+		<a class="navbar-brand" href="#">GAVIN WOOD</a>
+		<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup"
+			aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+		</button>
+
+		<div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+			<div class="navbar-nav">
+				<a class="nav-item nav-link active" href="/">Home</a>
+				<a class="nav-item nav-link" href="about">About</a>
+				<a class="nav-item nav-link" href="works">Works</a>
+			</div>
+		</div>
+	</nav>
+    <br>
+    <h3 style="text-align: center;">
+        Who is Gavin Wood?
+    </h3>
+    <br>
+    <ul class="list-unstyled">
+        <li>
+            Puede que alguna vez hayas escuchado hablar sobre Gavin Wood. Este es un programador informático británico conocido mayormente por ser el cofundador de Ethereum y el creador de Polkadot.
+        </li>
+        <li>
+            No solo es conocido por estos dos cargos. Wood también se encargó de inventar lo que hoy conocemos como Solidity y de la escritura del Yellow paper sobre la actual máquina virtual de Ethereum.
+        </li>
+        <br>
+        <li> <strong>Datos curiosos: </strong>
+          <ul>
+            <li>
+                Gavin fundó lo que conocemos como Fundación Web3.0. Esta es una fundación sin fines de lucro orientada a la infraestructura y tecnología de Internet.
+            </li>
+            <li>
+                Wood escribió las bases, la estructura de la blockchain, el Yellow Paper y el resto de especificaciones necesarias para dar forma a Ethereum
+            </li>
+          </ul>
+        </li>
+     </ul>
+    <br>
+    <br>
+    <br>
+    <div class="row row-cols-1 row-cols-md-2 g-4">
+        <div class="col">
+          <div style="border-color: grey;" class="card">
+            <img src="https://cryptoslate.com/wp-content/themes/cryptoslate-2020/imgresize/timthumb.php?src=https://cryptoslate.com/wp-content/uploads/2021/10/polkadot-defi-fund.jpg&w=600&h=315&q=75" class="card-img-top" alt="WOOD">
```

Comment:
Asi podemos hacer uso de las img que guardaste en tu carpeta 😄 
```suggestion
                     <img src="/images/Gavin-Wood.jpg" class="card-img-top" alt="WOOD">

```